src/components/backend-ai-tasker.ts

```diff
@@ -97,11 +96,11 @@ export default class BackendAiTasker extends LitElement {
    * @param {Object} task - Object / Promise instance to add.
    * @param {string} taskid - Task ID. It should be unique. If no taskid is given, it will be autogenerated (recommended)
    */
-  add(task, taskid: string = '') {
+  add(title, task, taskid: string = '', tasktype = 'general') {
     if (taskid === '') {
       taskid = this.generate_UUID();
     }
-    let item = new Task(task, taskid);
+    let item = new Task(title, task, taskid, tasktype);
     if (task != null && typeof task.then === 'function') { // For Promise type task
       task.then(() => {
           this.finished.push(taskid);
```

Comment:
Current implementation garbage collects finished tasks for 10 seconds period. So, a task that are finished in just 1 second stays in the list for 9 further seconds. What if garbage collect immediately here when a promise resolved, not waiting for 10 seconds for the GC cycle? Might it be too CPU consuming?

Reply:
When many tasks are finished together, it brings too many gc processes together. So I remove gc from original implementation (it was there after `finished.push`)

Reduce to 5 sec? It could be reasonable.

Reply:
Added responsive garbage collection.